packages/suite-web-landing/pages/index.tsx

```diff
@@ -31,39 +31,47 @@ const features = [
         headline: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_1_HEADLINE" />,
         text: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_1_TEXT" />,
         backgroundPosition: 'bottom right',
-        backgroundSize: '616px auto',
+        backgroundSize: 'auto 386px',
         soon: false,
     },
     {
         id: 2,
         headline: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_2_HEADLINE" />,
         text: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_2_TEXT" />,
         backgroundPosition: 'center left',
-        backgroundSize: '489px auto',
+        backgroundSize: 'auto 409px',
         soon: false,
     },
     {
         id: 3,
         headline: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_3_HEADLINE" />,
         text: <Translation id="TR_SUITE_WEB_LANDING_FEATURES_3_TEXT" />,
-        backgroundSize: '500px auto',
+        backgroundSize: 'auto 416px',
+        backgroundPosition: 'center right',
         soon: false,
     },
 ];
 
 const pathToApp = './web';
 
 const Index = () => {
+    const router = useRouter();
+
     const [translationMode, setTranslationMode] = useState(false);
+    const [isFromMyTrezor, setIsFromMyTrezor] = useState<boolean | undefined>(undefined);
 
-    const router = useRouter();
-    const { fromMytrezor } = router.query;
-    const isFromMytrezor = fromMytrezor === 'true';
+    useEffect(() => {
+        if (!router.isReady) return;
+
+        setIsFromMyTrezor(router.query?.fromMytrezor === 'true');
+    }, [router, setIsFromMyTrezor]);
 
     useEffect(() => {
         switchFavicon();
     }, []);
 
+    if (isFromMyTrezor === undefined) return null;
```

Comment:
This breaks the version served from server for everybody (including crawler bots) and enhance it just for those coming from old trezor wallet. I would prefer to find a way how to fix it server side (add another page for it?) or keep the default server version and just add some transition to make the flashing less annoying.

Reply:
Oh, true. Dropping the commit as it does not have priority.